libs/hbb_common/src/config.rs

```diff
@@ -1258,6 +1264,19 @@ serde_field_bool!(
     default_show_remote_cursor,
     "ShowRemoteCursor::default_show_remote_cursor"
 );
+serde_field_bool!(
+    FollowRemoteCursor,
+    "follow_remote_cursor",
+    default_follow_remote_cursor,
+    "FollowRemoteCursor::default_follow_remote_cursor"
```

Comment:
Default is true or false here?

Reply:
False

Reply:
Can you confirm the default value of "follow_remote_cursor" and "follow_focus_window" of AnyDesk?

Reply:
By default, both the features are off in AnyDesk.